Makefile

```diff
@@ -51,7 +51,6 @@ build-test-script:
 	echo 'export PYTHONPATH=$$script_dir/../$(LIB_DIR):$$PATH:$$PYTHONPATH' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'cd $$script_dir/../$(TEST_DIR)' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'python -m nose --with-coverage --cover-package=$(SERVICE_CAPS) --cover-html --cover-html-dir=/kb/module/work/test_coverage --cover-xml --cover-xml-file=/kb/module/work/test_coverage/coverage.xml --nocapture  --nologcapture .' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
-	echo 'cp /kb/module/.coveragerc .' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
```

Comment:
Why are this and the other line dealing with `.coveragerc` not needed?

Reply:
1. this file does not exist: https://github.com/kbaseapps/GenomeFileUtil/actions/runs/7404244710/job/20145462822#step:6:1100
2. Upload coverage to Codecov step works fine without it

Reply:
Interesting, it looks like when coverage was added to this repo no one added a coveragerc but did add the line: https://github.com/kbaseapps/GenomeFileUtil/commit/80056995d163a1a9b551dec79841ac6cae2ef6a8